python/nx-cugraph/nx_cugraph/convert.py

```diff
@@ -144,7 +142,7 @@ def from_networkx(
         else:
             node_attrs = {node_attrs: None}
 
-    if graph.__class__ in {nx.Graph, nx.DiGraph}:
+    if graph.__class__ in {nx.Graph, nx.DiGraph, nx.MultiGraph, nx.MultiDiGraph}:
```

Comment:
I'm assuming we can't use `isinstance()` here because this is accessing a private attr below?

Reply:
Correct, we want to be more conservative with subclasses.